Cogs/ServerManagement.py

```diff
@@ -339,3 +341,48 @@ async def on_button_click(self, res):
                 else:
                     await member.add_roles(role)
                     await res.respond(type=InteractionType.ChannelMessageWithSource, content=f'Gave you the {role.name} role!')
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        """
+        Checks if the author of the message was a bot and if it was sent in a correct channel
```

Comment:
I think this documentation can be made more precise.

Something like:
"Listens to every message on the server, responds if there's a '?' and the channel has FAQ enabled."

Reply:
This has been changed